docs/source/get_started/new_project.md

```diff
@@ -28,29 +27,91 @@ So, if you enter "Get Started", the folder for the project (`repo_name`) is auto
 
 ### Project tools
 
-The command line interface then asks which tools you'd like to include in the project. The options are as follows and described in more detail above in the [documentation about the new project tools](../starters/new_project_tools.md).
+Next, the CLI asks which tools you'd like to include in the project:
 
-You can add one or more of the options, or follow the default and add none at all:
+```text
+Which tools would you like to include in your project? [1-7/1,3/all/none]:
+
+Tools
+1) Lint: Basic linting with Black and Ruff
+2) Test: Basic testing with pytest
+3) Log: Additional, environment-specific logging options
+4) Docs: A Sphinx documentation setup
+5) Data Folder: A folder structure for data management
+6) PySpark: Configuration for working with PySpark
+7) Kedro-Viz: Kedro's native visualisation tool
+```
```

Comment:
This is the output I got with `main`:

````suggestion
```text
Tools
1) Lint: Basic linting with Black and Ruff
2) Test: Basic testing with pytest
3) Log: Additional, environment-specific logging options
4) Docs: A Sphinx documentation setup
5) Data Folder: A folder structure for data management
6) PySpark: Configuration for working with PySpark
7) Kedro-Viz: Kedro's native visualisation tool

Which tools would you like to include in your project? [1-7/1,3/all/none]:
 (none): 
```
````

skipped a few intro lines:

```
Project Tools
=============
These optional tools can help you apply software engineering best practices.
To skip this step in future use --tools
To find out more: https://docs.kedro.org/en/stable/starters/new_project_tools.html
```